fin_maestro_kin/modules/data_toolkit/screener/screener_equities.py

```diff
@@ -203,3 +203,44 @@ def get_ratios(
             return JSONResponse(content={"error": "Failed to fetch data from URL"}, status_code=500)
     except Exception as e:
         raise HTTPException(status_code=500, detail=f"Error fetching ratios data: {e}")
+
+
+def key_metrics(company_code):
+    url = f"https://www.screener.in/company/{company_code}/#top"
+    response = requests.get(url)
+
+    soup = BeautifulSoup(response.content, 'html.parser')
+    ratios = soup.find('div', {'class': 'company-ratios'})
+    ul_elements = ratios.find_all('ul')
+
+    if response.status_code == 200:
+        data = {}
+        for ul in ul_elements:
+            for li in ul.find_all('li'):
+                name = li.find('span', {'class': 'name'}).text.strip()
+                value = li.find('span', {'class': 'value'}).text.strip()
+                value = value.replace('\n', '').replace('\u20b9', '').replace('          ', '').replace('         ', '').replace('       ','').replace(' ','').replace(',','').replace('Cr.','Cr')
```

Comment:
Add a method to achieve in helper.py and call it here.

Reply:
Added a method in helper.py and called it here.